src/frontend/src/pages/AdminToolsPage/AdminToolsPage.tsx

```diff
@@ -91,6 +92,8 @@ const AdminToolsPage: React.FC = () => {
         <AdminToolsFinanceConfig />
       ) : tabIndex === 3 ? (
         <AdminToolsRecruitmentConfig />
+      ) : tabIndex === 4 ? (
+        <Box></Box>
```

Comment:
Make a config page similar to the others in this (even if you don't have anything for it yet)